src/collator/collator.py

```diff
@@ -65,6 +71,9 @@ def __init__(
     def reset_counter(self):
         self.samples_processed = 0
 
+    def update_epoch(self):
+        pass
```

Comment:
is this supposed to just be pass?

Reply:
Yep, only need the update epoch info for curriculum collator actually (so not the single task one and not the round robin one), just passing when the method is called on the other collators seemed like the easiest thing to do (not very elegant oc, so if you want to do something with it go ahead!)